modules/renter/persist.go

```diff
@@ -230,24 +241,42 @@ func (r *Renter) load() error {
 		}
 		return nil
 	})
-	if err != nil {
+}
+
+// load fetches the saved renter data from disk.
+func (r *Renter) loadSettings() error {
+	r.persist = persistence{
+		Tracking: make(map[string]trackedFile),
+	}
+	err := persist.LoadJSON(settingsMetadata, &r.persist, filepath.Join(r.persistDir, PersistFilename))
```

Comment:
no we get panics if we remove that line

Reply:
oh, that's probably because there's no file to load (the `os.IsNotExist` branch), so the field stays `nil`. The old persist code handled this like so:
https://github.com/NebulousLabs/Sia/blob/f9c05eca7c88c3191be9bb0752ca9028626107ad/modules/renter/persist.go#L237-L248

neither solution is very pretty, unfortunately. Initializing the field inside the struct declaration seems a bit cleaner though.